ajax/libs/jsSocials/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jsSocials",
+  "filename": "jssocials.min.js",
+  "description": "jsSocials - jQuery Social Network Sharing Plugin",
+  "homepage": "http://js-socials.com/",
+  "keywords": [
+    "jquery-plugin",
+    "social-network",
+    "sharing",
+    "social-share"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tabalinas/jssocials"
+  },
+  "license": "MIT",
+  "author": "Artem Tabalin <@tabalinas>",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/tabalinas/jssocials.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "jssocials*"
```

Comment:
Not sure if `**/*` is a better choice because we need every file in `dist` directory normally and we can not be sure that the author will change the naming rule of files in the future.